eZ/Publish/Core/MVC/Symfony/Controller/Content/ViewController.php

```diff
@@ -102,41 +99,9 @@ protected function buildResponse($etag = null, DateTime $lastModified = null)
      *
      * @deprecated Since 6.0.0. Viewing locations is now done with ViewContent.
      */
-    public function viewLocation($locationId, $viewType, $layout = false, array $params = array())
+    public function viewLocation(View $view)
```

Comment:
We can leave them like they were before. Typehinting the `View` would work, of course.